release-schema.json

```diff
@@ -65,53 +65,110 @@
+          "codelist": "financingArrangementType.csv",
           "openCodelist": false,
           "enum": [
-            "institutional",
-            "commercial",
-            "bilateral",
-            "multilateral",
-            "publicBondIssue",
-            "supplierCredit",
-            "other",
+            "bond",
+            "loan",
+            "hybrid",
+            "shares.listed",
+            "shares.unlisted",
+            "grant",
+            "guarantee",
             null
           ]
         },
-        "financeCategory": {
-          "title": "Finance category",
-          "description": "The nature of the finance, relevant to determining the order of preference for investors to get their money back in case of liquidation.",
+        "repaymentPriority": {
+          "title": "Repayment priority",
+          "description": "The priority in which the debt element of this financing arrangement will be repaid in the event of bankruptcy, from the open debtRepaymentPriority codelist.",
           "type": [
             "string",
             "null"
           ],
-          "codelist": "financeCategory.csv",
+          "codelist": "debtRepaymentPriority.csv",
           "openCodelist": false,
           "enum": [
-            "equity",
-            "seniorDebt",
-            "mezzanineDebt",
-            "grant",
-            "guarantee",
+            "senior",
+            "subordinated",
+            null
+          ]
+        },
+        "financingPartyType": {
+          "title": "Financing organization type",
+          "description": "The type of the financing organization, from the closed financingPartyType codelist.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "codelist": "financingPartyType.csv",
+          "openCodelist": false,
+          "enum": [
+            "institutional",
+            "commercial",
+            "bilateral",
+            "multilateral",
+            "vendor",
             "other",
             null
           ]
         },
+        "resultsBased": {
+          "title": "Results based",
+          "description": "Whether payments under this financing arrangement are contingent on the achievement of pre-defined results.",
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
         "value": {
           "title": "Value",
           "description": "The total value of this finance.",
           "$ref": "#/definitions/Value"
         },
         "period": {
           "title": "Finance period",
-          "description": "What period is covered by this finance.",
+          "description": "The period covered by this financing arrangement. Also called maturity.",
+          "$ref": "#/definitions/Period"
+        },
+        "repaymentPeriod": {
+          "title": "Repayment period",
+          "description": "The period over which the borrower or issuer for this financing arrangement is obliged to make repayments.",
```

Comment:
In what scenario does the issuer make repayments?

Reply:
Governments and companies issue bonds, in which case they are typically referred to as issuers rather than borrowers.

However, https://www.investopedia.com/terms/b/bond.asp describes bond issuers as borrowers in the first paragraph so we could remove the reference to issuers if you think it's confusing.

Looking at it again, I wonder if we should also rename `repaymentPeriod` to `paymentPeriod` to better suit bonds, where the only payments made before the maturity date are interest payments (rather than repayments of the principal).

Reply:
`paymentPeriod` sounds good to me.

Reply:
Done in https://github.com/open-contracting-extensions/ocds_finance_extension/pull/26/commits/6aed191dc840c021cf5dda80167540146bc72a28

Reply:
Can also then change "repaid" to "paid".